tests/php/Security/MemberTest.php

```diff
@@ -1571,4 +1571,18 @@ public function testNewMembersReceiveTheDefaultLocale()
 
         $this->assertSame('en_US', $newMember->Locale, 'New members receive the default locale');
     }
+
+    public function testChangePasswordHashesPasswordsOnce()
```

Comment:
```suggestion
    public function testChangePasswordOnlyValidatesPlaintext()
```

Reply:
A test for ensuring passwords only hash once isn't a bad idea too 😉

Reply:
I think if the system were hashing the password twice, existing tests would pick that up